Program does nothing when clicking on a container in the containers tab

I just re-downloaded and builded mysef the program and got this :
```
Exception in thread "JavaFX Application Thread" java.lang.IllegalStateException: java.lang.NoSuchMethodException: org.phoenicis.javafx.views.mainwindow.containers.WinePrefixContainerPanel.<init>(org.phoenicis.containers.dto.WinePrefixContainerDTO, java.util.List, org.phoenicis.engines.EngineToolsManager, java.util.Optional, org.phoenicis.containers.wine.WinePrefixContainerController)
        at org.phoenicis.javafx.views.mainwindow.containers.ContainerPanelFactory.createContainerPanel(ContainerPanelFactory.java:53)
        at org.phoenicis.javafx.controller.containers.ContainersController.lambda$new$8(ContainersController.java:80)
        at org.phoenicis.javafx.views.mainwindow.containers.ContainersView.showContainerDetails(ContainersView.java:130)
        at org.phoenicis.javafx.views.mainwindow.containers.ContainersView.lambda$new$0(ContainersView.java:77)
        at org.phoenicis.javafx.views.common.widgets.lists.CombinedListWidget.lambda$new$0(CombinedListWidget.java:67)
        at org.phoenicis.javafx.views.common.widgets.lists.icons.IconsListWidget.lambda$null$0(IconsListWidget.java:84)
        at com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(CompositeEventHandler.java:86)
        at com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:238)
        at com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:191)
        at com.sun.javafx.event.CompositeEventDispatcher.dispatchBubblingEvent(CompositeEventDispatcher.java:59)
        at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:58)
        at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
        at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
        at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
        at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
        at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
        at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
        at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
        at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
        at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
        at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
        at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
        at com.sun.javafx.event.EventUtil.fireEventImpl(EventUtil.java:74)
        at com.sun.javafx.event.EventUtil.fireEvent(EventUtil.java:54)
        at javafx.event.Event.fireEvent(Event.java:198)
        at javafx.scene.Scene$ClickGenerator.postProcess(Scene.java:3470)
        at javafx.scene.Scene$ClickGenerator.access$8100(Scene.java:3398)
        at javafx.scene.Scene$MouseHandler.process(Scene.java:3766)
        at javafx.scene.Scene$MouseHandler.access$1500(Scene.java:3485)
        at javafx.scene.Scene.impl_processMouseEvent(Scene.java:1762)
        at javafx.scene.Scene$ScenePeerListener.mouseEvent(Scene.java:2494)
        at com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(GlassViewEventHandler.java:381)
        at com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(GlassViewEventHandler.java:295)
        at java.security.AccessController.doPrivileged(Native Method)
        at com.sun.javafx.tk.quantum.GlassViewEventHandler.lambda$handleMouseEvent$2(GlassViewEventHandler.java:417)
        at com.sun.javafx.tk.quantum.QuantumToolkit.runWithoutRenderLock(QuantumToolkit.java:389)
        at com.sun.javafx.tk.quantum.GlassViewEventHandler.handleMouseEvent(GlassViewEventHandler.java:416)
        at com.sun.glass.ui.View.handleMouseEvent(View.java:555)
        at com.sun.glass.ui.View.notifyMouse(View.java:937)
        at com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
        at com.sun.glass.ui.gtk.GtkApplication.lambda$null$5(GtkApplication.java:139)
        at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NoSuchMethodException: org.phoenicis.javafx.views.mainwindow.containers.WinePrefixContainerPanel.<init>(org.phoenicis.containers.dto.WinePrefixContainerDTO, java.util.List, org.phoenicis.engines.EngineToolsManager, java.util.Optional, org.phoenicis.containers.wine.WinePrefixContainerController)
        at java.lang.Class.getConstructor0(Class.java:3082)
        at java.lang.Class.getConstructor(Class.java:1825)
        at org.phoenicis.javafx.views.mainwindow.containers.ContainerPanelFactory.createContainerPanel(ContainerPanelFactory.java:47)
        ... 41 more
```
Also have some 
`[DEBUG] org.phoenicis.library.LibraryManager (l.157) - JSON file not found
`

--- a/phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/containers/ContainersController.java
+++ b/phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/containers/ContainersController.java
@@ -33,7 +33,6 @@ import org.phoenicis.repository.RepositoryManager;
 import org.phoenicis.repository.dto.ApplicationDTO;
 import org.phoenicis.repository.dto.RepositoryDTO;
 
-import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Optional;
@@ -79,7 +78,6 @@ public class ContainersController {
             // enginesSource.fetchAvailableEngines(categoryDTOS, engineCategoryDTOS -> {
             final WinePrefixContainerPanel panel = winePrefixContainerPanelFactory.createContainerPanel(
                     (WinePrefixContainerDTO) containerDTO,
-                    new ArrayList<>(),
                     /*engineCategoryDTOS.stream().flatMap(category -> category.getSubCategories().stream())
                     .flatMap(subCategory -> subCategory.getPackages().stream())
                     .collect(Collectors.toList()),*/

--- a/phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/ContainerPanelFactory.java
+++ b/phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/ContainerPanelFactory.java
@@ -21,11 +21,9 @@ package org.phoenicis.javafx.views.mainwindow.containers;
 import org.phoenicis.containers.dto.ContainerDTO;
 import org.phoenicis.containers.wine.WinePrefixContainerController;
 import org.phoenicis.engines.EngineToolsManager;
-import org.phoenicis.engines.dto.EngineVersionDTO;
 import org.phoenicis.repository.dto.ApplicationDTO;
 
 import java.lang.reflect.InvocationTargetException;
-import java.util.List;
 import java.util.Optional;
 
 public class ContainerPanelFactory<T extends AbstractContainerPanel<C>, C extends ContainerDTO> {
@@ -38,15 +36,14 @@ public class ContainerPanelFactory<T extends AbstractContainerPanel<C>, C extend
     }
 
     public T createContainerPanel(C containerDTO,
-            List<EngineVersionDTO> engineVersions,
             EngineToolsManager engineToolsManager,
             Optional<ApplicationDTO> engineTools,
             WinePrefixContainerController winePrefixContainerController) {
         try {
             return this.clazz
-                    .getConstructor(entityClazz, List.class, EngineToolsManager.class, Optional.class,
+                    .getConstructor(entityClazz, EngineToolsManager.class, Optional.class,
                             WinePrefixContainerController.class)
-                    .newInstance(containerDTO, engineVersions, engineToolsManager, engineTools,
+                    .newInstance(containerDTO, engineToolsManager, engineTools,
                             winePrefixContainerController);
         } catch (InstantiationException | IllegalAccessException | InvocationTargetException
                 | NoSuchMethodException e) {

